frontend/src/components/Module/ModuleDetailsRejectEvents.vue

```diff
@@ -26,49 +26,23 @@
 			</TableTd>
 			<TableTd>
 				{{ moduleRejectEvent.rejectedEvent.__typename }}
+				<InfoTooltip :text="getEventTooltip(moduleRejectEvent.rejectedEvent.__typename!)"/>
 			</TableTd>
 			<TableTd>
-				<div
-					v-if="
-						moduleRejectEvent.rejectedEvent.__typename === 'InvalidInitMethod'
-					"
-				>
-					<div>Init Name:</div>
-					<div>{{ moduleRejectEvent.rejectedEvent.initName }}</div>
-					<div>Module Reference:</div>
-					<div>
-						<ModuleLink
-							:module-reference="moduleRejectEvent.rejectedEvent.moduleRef"
-						/>
+				<div v-if="moduleRejectEvent.rejectedEvent.__typename === 'InvalidInitMethod'">
+					<div>Init name:
+						<InfoTooltip text="Initial entrypoint of the contract."/>
 					</div>
+					<div>{{ moduleRejectEvent.rejectedEvent.initName }}</div>
 				</div>
-				<div
-					v-if="
-						moduleRejectEvent.rejectedEvent.__typename ===
-						'InvalidReceiveMethod'
-					"
-				>
-					<div>Receive Name:</div>
-					<div>{{ moduleRejectEvent.rejectedEvent.receiveName }}</div>
-					<div>Module Reference:</div>
-					<div>
-						<ModuleLink
-							:module-reference="moduleRejectEvent.rejectedEvent.moduleRef"
-						/>
+				<div v-if="moduleRejectEvent.rejectedEvent.__typename === 'InvalidReceiveMethod'">
+					<div>Receive name:
+						<InfoTooltip text="Entrypoint of the activity of the contract."/>
```

Comment:
I don't understand where this "activity" terminology is coming from. I would say just "entrypoint of the contract instance".